src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -100,6 +103,9 @@ public async Task GetDocSize()
                 await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
                 {
                     context.Write(writer, documentSizeDetails.ToJson());
+                    
+                    if (TrafficWatchManager.HasRegisteredClients)
```

Comment:
we are returning earlier in this method on `if (document == null)`, and we still need to log

I wondering if we can do:
```
if (TrafficWatchManager.HasRegisteredClients)
   AddStringToHttpContext("N/A", TrafficWatchChangeType.Documents);
```

In the constructor of this class so in case of exception we still will log in under `Documents` category

Reply:
Done